File: src/main/java/ru/trukhmanov/twochairsbackend/repository/ProductRepository.java
package ru.trukhmanov.twochairsbackend.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import ru.trukhmanov.twochairsbackend.entity.Product;

import java.util.List;

public interface ProductRepository extends JpaRepository<Product, Long> {
    List<Product> findByActiveTrueOrderByIdAsc();
}

File: src/main/kotlin/ru/trukhmanov/twochairsbackend/repository/ProductRepository.kt
package ru.trukhmanov.twochairsbackend.repository

import org.springframework.data.jpa.repository.JpaRepository
import ru.trukhmanov.twochairsbackend.entity.Product

interface ProductRepository : JpaRepository<Product, Long> {
    fun findByActiveTrueOrderByIdAsc(): List<Product>
}
